framework/src/modules/poa/endpoint.ts

```diff
@@ -30,10 +32,10 @@ export class PoAEndpoint extends BaseEndpoint {
 
 	public async getValidator(context: ModuleEndpointContext): Promise<Validator> {
 		const validatorSubStore = this.stores.get(ValidatorStore);
-		const { address } = context.params;
-		if (typeof address !== 'string') {
-			throw new Error('Parameter address must be a string.');
-		}
+		const address = context.params.address as string;
+
+		validator.validate(getValidatorRequestSchema, context.params);
```

Comment:
I think validate should be called before we access address param
```suggestion
                validator.validate(getValidatorRequestSchema, context.params);
                
		const address = context.params.address as string;
```